test/catalogue.test.js

```diff
@@ -210,19 +213,40 @@ describe('Catalogue', () => {
         it('should send empty columns if only date buckets are in the request', () => {
             const mockPayload = {
```

Comment:
You might consider moving these large objects to a new fixtures file.

Reply:
moved to existing fixtures file